pkg/querier/frontend/frontend.go

```diff
@@ -152,27 +155,26 @@ func (f *Frontend) Handler() http.Handler {
 }
 
 func (f *Frontend) handle(w http.ResponseWriter, r *http.Request) {
-	userID, err := user.ExtractOrgID(r.Context())
-	if err != nil {
-		server.WriteError(w, err)
-		return
-	}
 
 	startTime := time.Now()
 	resp, err := f.roundTripper.RoundTrip(r)
 	queryResponseTime := time.Since(startTime)
 
 	if f.cfg.LogQueriesLongerThan > 0 && queryResponseTime > f.cfg.LogQueriesLongerThan {
-		logMessage := []interface{}{"msg", "slow query",
-			"org_id", userID,
-			"url", fmt.Sprintf("http://%s", r.Host+r.RequestURI),
+		logMessage := []interface{}{
+			"msg", "slow query",
+			"host", r.Host,
+			"path", r.URL.Path,
 			"time_taken", queryResponseTime.String(),
 		}
+		for k, v := range r.URL.Query() {
+			append(logMessage, fmt.Sprintf("qs_%s", k), strings.Join(v, ","))
```

Comment:
There's a bug here. `logMessage` is not updated with the result of `append()`. I fix it for you.